cli/bin/setup

```diff
@@ -7,15 +7,4 @@ if [ $? -gt 0 ]; then
   exit 1
 fi
 yarn install
-(cd ../exo-add && bin/setup && echo 'exo-add √')
-(cd ../exo-clone && bin/setup && echo 'exo-clone √')
-(cd ../exo-create && bin/setup && echo 'exo-create √')
-(cd ../exo-deploy && bin/setup && echo 'exo-deploy √')
-(cd ../exo-lint && bin/setup && echo 'exo-lint √')
-(cd ../exo-run && bin/setup && echo 'exo-run √')
-(cd ../exo-setup && bin/setup && echo 'exo-setup √')
-(cd ../exo-sync && bin/setup && echo 'exo-sync √')
-(cd ../exo-test && bin/setup && echo 'exo-test √')
-(cd ../exosphere-shared && bin/setup && echo 'exosphere-shared √')
-(node_modules/.bin/build)
-(./bin/build)
+node_modules/.bin/build
```

Comment:
This repo now only prepares itself, and expects the other repos to be set up as well. The idea here is that if one changes code in this repo, it should only recompile this repo. I'm not 100% sure on this, open for feedback.